.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+sudo: false
+dist: trusty
+python: "3.5"
+
+cache:
+    apt: true
+    pip: true
+    ccache: true
+
+before_install:
+    - pip install -U setuptools pip wheel
+    - pip install codecov
+
+install:
+    - pip install PyQt5 AnyQt
+    - travis_wait pip install -e .
+
+script:
+    - catchsegv xvfb-run -a -s "-screen 0 1280x1024x24" coverage run setup.py test
```

Comment:
AFAIU, we have zero tests and this actually does nothing?

Reply:
Tests will be added in a separate PR.